src/insert/spacefinder/article-body-adverts.ts

```diff
@@ -290,16 +290,25 @@ const addDesktopInline2PlusAds = (): Promise<boolean> => {
 const addMobileInlineAds = (): Promise<boolean> => {
 	const rules: SpacefinderRules = {
 		bodySelector: articleBodySelector,
-		slotSelector: ' > p',
+		slotSelector: ' > p,hr,h2',
 		minAbove: 200,
 		minBelow: 200,
 		selectors: {
 			' > h2': {
-				minAbove: 100,
+				minAbove: 0,
 				minBelow: 250,
```

Comment:
I think I might be misunderstand this line. I assumed that this means that we can't insert an ad below a h2 within 250px of it. However, in this article we clearly do not have 250px of space between the h2 and the advert. The paragraph between the h2 and the advert is about 119px tall:

<img width="400" alt="image" src="https://github.com/guardian/commercial/assets/9574885/5a67125e-de6e-4def-9f19-fb03479ebdbd">

https://www.theguardian.com/music/2024/feb/22/bliss-out-strip-off-or-headbang-your-bumper-guide-to-the-35-best-music-festivals-in-europe